packages/contracts/src/dollar/core/StakingShare.sol

```diff
@@ -116,58 +113,10 @@ contract StakingShareForDiamond is ERC1155, ERC1155Burnable, ERC1155Pausable {
         _totalLP += lpDeposited;
     }
 
-    /**
-     * @dev Pauses all token transfers.
-     *
-     * See {ERC1155Pausable} and {Pausable-_pause}.
-     *
-     */
-    function pause() public virtual onlyPauser {
-        _pause();
-    }
-
-    /**
-     * @dev Unpauses all token transfers.
-     *
-     * See {ERC1155Pausable} and {Pausable-_unpause}.
-     *
-     */
-    function unpause() public virtual onlyPauser {
-        _unpause();
-    }
-
-    /**
-     * @dev See {IERC1155-safeTransferFrom}.
-     */
-    function safeTransferFrom(
-        address from,
-        address to,
-        uint256 id,
-        uint256 amount,
-        bytes memory data
-    ) public override whenNotPaused {
-        super.safeTransferFrom(from, to, id, amount, data);
-        _holderBalances[to].add(id);
-    }
-
-    /**
-     * @dev See {IERC1155-safeBatchTransferFrom}.
-     */
-    function safeBatchTransferFrom(
-        address from,
-        address to,
-        uint256[] memory ids,
-        uint256[] memory amounts,
-        bytes memory data
-    ) public virtual override whenNotPaused {
-        super.safeBatchTransferFrom(from, to, ids, amounts, data);
-        _holderBalances[to].add(ids);
-    }
-
```

Comment:
Seems like a lot of functions were removed. I presume that they were moved to the inherited Ubiquity ERC 1155 file.

Reply:
yep